modules/ROOT/pages/box/box-connector-examples.adoc

```diff
@@ -146,35 +146,36 @@ The corresponding XML configuration should be as follows:
 |Folder id|`#[payload.id]`
 |===
 +
-. Then, drag the DataWeave component to set the payload to display the result of the flow to the user and use the following script.
+. Drag the DataWeave component to set the payload to display the result of the flow to the user, and use the following script:
 +
 [source,dataweave,linenums]
 ----
 %dw 2.0
 output application/json
 ---
 {
-	result : "Folder got created, retrieved and deleted successfully"
+	result : "Folder was created, retrieved and deleted successfully."
```

Comment:
This is output? I'm not sure we should be changing wording here if it's actual output. I guess we need to fix the source in this case.